syntax = "proto3";

message EventClockPhase {}
message EventMainPhase {}
message EventClimaxPhase {}
message EventAttackDeclarationStep {}
message EventEndOfAttack {}
message EventCounterStep {}
message EventEncoreStep {}
message EventPhaseEvent {
    sint32 phase = 1;
}
message EventEndOfPhase {}
